tensorflow/go/session_test.go

```diff
@@ -375,8 +375,8 @@ func TestSessionConfig(t *testing.T) {
 	// tensorflow Python program:
 	/*
 	 import tensorflow
-	 c = tensorflow.ConfigProto()
-	 c.intra_op_parallelism_threads = 1
+	 c = tensorflow.compat.v1.ConfigProto()
```

Comment:
We should not use `v1` symbols in documentation anymore, unless it explicitly refers to v1 code.

Reply:
Ok, is there an alternate path to the ConfigProto?

Or even better would be to use the golang protobuf code to get the "config" bytes example in this test. 
I could maybe do that if you think it is a good idea.

I just think that, since the comment is outdated, it makes sense to either update it or remove it, right?

Reply:
Oh, the comment is in a v1 API anyway.